recipe/meta.yaml

```diff
@@ -18,10 +18,11 @@ build:
 requirements:
   build:
     # dependencies needed for open source microsoft/vscode - https://github.com/microsoft/vscode/wiki/How-to-Contribute 
-    - nodejs >=18.15,<19
+    - nodejs >=20.9
     - yarn <2
     - make
     - pkg-config
+    - compilers
```

Comment:
good to add comment on why this is needed ( that way we know if it can be removed in future )